variables.tf

```
@@ -295,12 +295,25 @@ variable "enable_logging" {
   default     = false
 }
 
-variable "deploy_athena_queries" {
-  description = "Enables the deployment of the athena pre-saved queries to easily access the logs generated by waf"
+
+variable "deploy_logs_bucket" {
```

```suggestion
variable "deploy_logs_bucket_and_queries" {
```
just to make it clear what is deployed in the module. In reality, if would prefer to keep both variable

---

In that context they usually come together no ? You don't need athena queries if you don't have a bucket to browse 🤔

---

Done